components/brave_wallet_ui/page/screens/swap/adapters.ts

```diff
@@ -296,10 +301,19 @@ export function makeSOLSendTransaction (
   return sendTransaction
 }
 
-export function makeSwitchAccount () {
-  async function switchAccount (account: WalletAccount) {
+export function makeSwitchAccount(accounts: WalletAccountType[]) {
+  async function switchAccount(account: WalletAccount) {
     const { keyringService } = getAPIProxy()
-    await keyringService.setSelectedAccount(account.address, account.coin)
+
+    // TODO(apaymyshev): swap's WalletAccount should somehow provide
+    // corresponding instance of BraveWallet.AccountInfo
```

Comment:
Let's create and attach a follow-up issue for this. I believe we want to move away from using `WalletAccountType` over `BraveWallet.AccountInfo`